plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/SpringPlugin.java

```diff
@@ -79,6 +82,9 @@
                 PostProcessorRegistrationDelegateTransformer.class,
                 BeanFactoryTransformer.class,
                 InitDestroyAnnotationBeanPostProcessorTransformer.class})
+@Versions(manifest = {@Manifest(value = "[3.0,6.0)", versionName= Name.ImplementationVersion, names={
```

Comment:
Maybe it should start at 3.1

Reply:
done. I set it starting from 3.1.0 without ending. (for 6.x, org.hotswap.agent.plugin.spring.getbean.ProxyReplacer#register is not compatible)